flashlight/nn/modules/Module.cpp

```diff
@@ -88,6 +90,25 @@ Variable UnaryModule::operator()(const Variable& input) {
   return this->forward(input);
 }
 
+PrecisionCast::PrecisionCast(af::dtype targetType) : targetType_(targetType) {}
```

Comment:
Can we move `PrecisionCast` to its own file rather than defining it in `Module`?

Reply:
Done.